Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:7.6.1810
+
+RUN yum update -y && \
```

Comment:
If we're building images we'll subsequently tag I'm not sure it's worth giving super specific versions. We may as well pick up any updates by keeping the package versions loose

Reply:
Done.